features/proxy_config.feature

```diff
@@ -13,41 +13,35 @@ Feature: Proxy configuration
             dns_v4_first on\nacl all src 0.0.0.0\/0\nhttp_access allow all
             """
         And I run `systemctl restart squid.service` `with sudo` on the `proxy` machine
-        When I create the file `/etc/ubuntu-advantage/uaclient.conf` with the following:
+        And I run `ua config set https_proxy=http://<ci-proxy-ip>:3128` with sudo
+        Then stdout matches regexp:
         """
-        contract_url: 'https://contracts.canonical.com'
-        data_dir: /var/lib/ubuntu-advantage
-        log_level: debug
-        log_file: /var/log/ubuntu-advantage.log
-        ua_config:
-          http_proxy: http://<ci-proxy-ip>:3128
-          https_proxy: http://<ci-proxy-ip>:3128
```

Comment:
I appreciate that this change uses the recommended way of setting the proxy now (`ua config set`), but now there is no integration test that tests the proxies are correctly configured after editing the config file and running `ua refresh`.

We can keep these changes, but we may need a new scenario to test setting the proxy via config file and `ua refresh config` also works so that we don't lose this coverage.

Reply:
Yeah, I think adding a new scenario would be helpful to cover this test.